packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config-editor.vue

```diff
@@ -116,18 +204,25 @@
 
 <script setup lang="ts">
 import _ from 'lodash';
-import { computed, ref, watch } from 'vue';
-import Accordion from 'primevue/accordion';
-import AccordionTab from 'primevue/accordiontab';
+import { computed, ref, Ref, watch } from 'vue';
 import Button from 'primevue/button';
 import InputNumber from 'primevue/inputnumber';
 import InputText from 'primevue/inputtext';
-import { Model, Initial, ModelParameter } from '@/types/Types';
+import type { Model, ModelConfiguration, ModelParameter, Initial } from '@/types/Types';
 import {
 	getUnstratifiedParameters,
 	getUnstratifiedInitials
 } from '@/model-representation/petrinet/mira-petri';
 import { getStratificationType } from '@/model-representation/petrinet/petrinet-service';
+import TeraStratifiedMatrix from '@/components/model/petrinet/model-configurations/tera-stratified-matrix.vue';
+import { StratifiedMatrix } from '@/types/Model';
+
+enum FieldTypes {
+	Parameter = 'parameter',
+	Initial = 'initial',
+	StratifiedParameter = 'stratified-parameter',
+	StratifiedInitial = 'stratified-initial'
+}
 
 const props = defineProps<{
```

Comment:
Can you add a explanation comment here? Because a model already has initials and parameters, why do we need to pass them in separately?

Reply:
these initials and parameters are the ones that are stored in state: https://github.com/DARPA-ASKEM/terarium/blob/bd5323b5a071de2f614fb7c4db602253168bce48/packages/client/hmi-client/src/workflow/ops/model-config/model-config-operation.ts#L10-L11

I thought it would be better to just store the initials/parameters in state instead of the whole model. This also makes it a bit easier (I think) to modify and track the fields in the wizard as well as switch between different outputs.